src/client/js/partials/all-breaches.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const allBreachesPartial = document.querySelector('[data-partial="allBreachesPartial"]')
+
+if (allBreachesPartial) {
+  const search = document.getElementById('breach-search')
```

Comment:
micro-optimization nit!  Once we confirm `allBreachesPartial` we can henceforth query within that element instead of the entire document, e.g.  `allBreachesPartial.getElementById('breach-search')` and `allBreachesPartial.querySelectorAll('.breach-card')`. 

I'll see myself out.